### PR TITLE
No javascript phase one

### PR DESCRIPTION
**Pull request by borislavpetrovhmcts**
# Description

Insert tracking image for detecting users with disabled javascript

Fixes # DIV-4169

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Tested locally

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -58,6 +58,10 @@ lookAndFeel.configure(app, {
 // Get user details from idam, sets req.idam.userDetails
 app.use(idam.userDetails());
 
+app.get('/assets/images/noJS.png', (req, res) => {
```
Comment (dtaylor7) [inline app.js]: you can use any endpoint here, so no need for `/assets/images/noJS.png` just use `noJS.png`

File: app.js [hunk 2]
```diff
@@ -58,6 +58,10 @@ lookAndFeel.configure(app, {
...
+  res.send('data:image/gif;base64,R0lGODlhAQABAAAAACH5BAEKAAEALAAAAAABAAEAAAICTAEAOw==');
```
Comment (dtaylor7) [inline app.js]: insert comment to let people know this is a 1x1 pixel image

File: views/includes/problem_with_this_page.html [hunk 3]
```diff
@@ -11,4 +11,7 @@
         <p class="text">{{ content.isThereAProblemWithThisPageEmail }}</p>
       </div>
     </details>
-  </div>
\ No newline at end of file
+  </div>
+  <noscript>
```
Comment (dtaylor7) [inline views/includes/problem_with_this_page.html]: this should be at the top of the page, ideally one of the first elements within the body tag

File: app.js (hunk 2 above)
Comment (borislavpetrovhmcts) [reply app.js]: done

File: app.js (hunk 1 above)
Comment (borislavpetrovhmcts) [reply app.js]: updated

File: views/includes/problem_with_this_page.html (hunk 3 above)
Comment (borislavpetrovhmcts) [reply views/includes/problem_with_this_page.html]: updated

File: app.js (hunk 2 above)
Comment (ravinderpal) [reply app.js]: what is the use of this 1x1 px image?

**Pull request merged**